README.rst

```diff
@@ -41,6 +41,27 @@ Install with pip
 
    pip install python-gitlab
 
+
+Using the python-gitlab docker image
+====================================
+
+How to build
+------------
+
+``docker build -t python-gitlab:TAG .``
+
+How to use
+----------
+
+``docker run -it --rm  -e GITLAB_PRIVATE_TOKEN=<your token> =/python-gitlab.cfg python-gitlab <command> ...``
```

Comment:
Ghost ` =/python-gitlab.cfg` ?

Reply:
I copied that from contrib/docker/README.rst and just glanced over it. Nice find.